README.md

```diff
@@ -153,6 +153,11 @@ Will generate:
 * `getMediaMimeTypesCaption(string $collectionName): string`: returns only the mime types caption for the given collection.
 * `getMediaSizeCaption(): string`: returns only the config max file size caption only.
 
+###Exceptions
+
+In order to avoid careless mistakes an exception will be thrown when the specified collection is not found in the 
```

Comment:
```suggestion
In order to avoid careless mistakes a `Okipa\MediaLibraryExt\Exceptions\CollectionNotFound` exception will be thrown when the specified collection is not found in the 
```